File: Lat2/Python/Main.py
from Product import Product
from Hardware import Hardware
from Memory import Memory

gforce = Memory()
gforce.setBrand("Nvidia")
gforce.setModel("GPU")
gforce.setFrequency("8_GBPS")
gforce.setCuda("Yes")
gforce.setSize("8_GB")
gforce.setId("gtx-1650")
gforce.setPrice("RP_5.200.000")
print("Brand Memory: " + str(gforce.getBrand()))
print("Model Memory: " + str(gforce.getModel()))
print("Frequency Memory: " + str(gforce.getFrequency()))
print("SupportCuda Memory: " + str(gforce.getCuda()))
print("Memory Size: " + str(gforce.getSize()))
print("ID Memory: " + str(gforce.getId()))
print("Price Memory: " + str(gforce.getPrice()))

File: Lat2/C++/Main.cpp
#include <iostream>
#include <string>
using namespace std;
#include "Memory.cpp"

int main (){
    Memory tforce;
    tforce.setBrand("Nvidia");
    tforce.setModel("GPU");
    tforce.setFrequency("8_GBPS");
    tforce.setCuda("Yes");
    tforce.setSize("8_GB");
    tforce.setId("gtx-1650");
    tforce.setPrice("RP_5.200.000");
    cout << "Brand Memory: "<<tforce.getBrand() << endl;
    cout << "Model Memory: "<<tforce.getModel() << endl;
    cout << "Frequency Memory: "<< tforce.getFrequency() << endl;
    cout << "SupportCuda Memory: "<< tforce.getCuda() << endl;
    cout << "Memory Size: "<< tforce.getSize() << endl;
    cout << "ID Memory: " << tforce.getId() << endl;
    cout << "Price Memory: " << tforce.getPrice() << endl;
    return 0;
}